environments/_users/dimagi.yml

```diff
@@ -23,6 +23,7 @@ dev_users:
     - sbansal
     - sherbers
     - spokhriyal
+    - matthewriley
```

Comment:
two nits:
all other users are first initial last name
all other users are alphabetized

the second is probably more important than the first but it is nice to keep the username convention

Reply:
@calellowitz i figured the username thing would come up... `mattewriley` is his local username and it would make ssh so much easier for him

Reply:
changed the order tho, that's fair

Reply:
also, not everyone's username is exactly first initial last name 👀

Reply:
you and sravan always breaking things 😛 , but yeah. its a one line ssh config change to make it handle that automatically, and i think conventions are nice. but its obviously not a big deal and still clear who that user is